src/cli/argument-parser.js

```diff
@@ -227,11 +228,12 @@ export default class CLIArgumentParser {
     }
 
     async parse (argv) {
-        this.program.parse(argv);
+        const { args, v8Flags } = extractNodeProcessArguments(argv);
 
-        this.args = this.program.args;
+        this.program.parse(args);
 
-        this.opts = this.program.opts();
+        this.args = this.program.args;
+        this.opts = Object.assign(this.program.opts(), { v8Flags });
```

Comment:
Why do we need to append V8 flags to `this.opts` property?

Reply:
because we need to pass `v8Flags` deeper to the runner class, and then to the bootstrapper and compiler